### ISSUE TITLE
Sink record timestamp as version number

### ISSUE THREAD

**Issue by patsak** [confluentinc/kafka-connect-elasticsearch#81]
Hi! 
Sometimes it's useful to take timestamp from kafka records as version number instead partition offset.
For example when you add partitions in a target topic.

**Comment by davidepozza** [comment]
I agree, it's useful also if you need to make ES totally independent from KAFKA topics: by using the partition number KAFKA acts as a primary storage and for example you cannot empty/delete any source topic.
I had to patch the connector in order to use the record timestamp: it seems to work well on my scenario (time series storage)
